From c95f881ede04208295bed6b661d52dd8e909db0a Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Tue, 6 Mar 2018 15:37:10 +0800
Subject: [PATCH 3447/5242] MLK-17698-1 thermal: imx_sc: use system controller
 thermal sensor for A35 CPU

commit  2f8723943441af5bcf2705cc6498732faa0f56f7 from
https://source.codeaurora.org/external/imx/linux-imx.git

Now that SCFW (0d43db9 SCF-22: Move SCU controls to SYSTEM.
Allows AP to use SCU temp sensor.) exposes SCU's temp sensor
for AP, and it is placed more close to i.MX8QXP A35 core, so
it should be used as A35's CPU thermal sensor, add this change
and move DRC temp sensor to a new thermal zone.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Reviewed-by: Bai Ping <ping.bai@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/thermal/imx_sc_thermal.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/thermal/imx_sc_thermal.c b/drivers/thermal/imx_sc_thermal.c
index eb4c535..2a3aad7 100644
--- a/drivers/thermal/imx_sc_thermal.c
+++ b/drivers/thermal/imx_sc_thermal.c
@@ -1,5 +1,5 @@
 /*
- * Copyright 2017 NXP.
+ * Copyright 2017-2018 NXP.
  *
  * This program is free software; you can redistribute it and/or modify
  * it under the terms of the GNU General Public License version 2 as
@@ -49,7 +49,7 @@ enum imx_thermal_trip {
 };
 
 static const sc_rsrc_t imx8qxp_sensor_hw_id[] = {
-	SC_R_DRC_0,
+	SC_R_SYSTEM, SC_R_DRC_0,
 };
 
 const int *sensor_hw_id;
-- 
1.7.9.5

